src/adr/0039-distributed-blob-garbage-collector.md

```diff
@@ -58,6 +60,10 @@ store the blob for. This can make some components harder to implement as we will
 Since we will not build a distributed task scheduler. To increment the `reference generation` and launch periodically a
 `garbage collection iteration`, the scheduling will be done by an external scheduler (cron job, kubernetes cronjob ...)
  which will call a webadmin endpoint to launch this task periodically.
+ 
+## References
+
+
```

Comment:
empty references?

Reply:
Likely I found nothing!

Maybe implied actors @remk @rouazana could be helping here?

Reply:
https://issues.apache.org/jira/browse/JAMES-3150 ?
And probably the PR creating this ADR?

Reply:
> And probably the PR creating this ADR?

Which is? Sorry, I did not find it!

Reply:
https://github.com/linagora/james-project/pull/3122

probably a mistake to not have put it on the Apache repo